weldx/transformations/local_cs.py

```diff
@@ -303,6 +305,26 @@ def _build_coordinates(coordinates, time: pd.DatetimeIndex = None):
         xarray.DataArray
 
         """
+        if isinstance(coordinates, TimeSeries):
+            if coordinates.is_expression:
+                return coordinates
+            if coordinates.shape[1] != 3:
+                raise ValueError(
+                    "The shape of the TimeSeries must be (n, 3). It actually is: "
+                    f"{coordinates.shape}"
+                )
+            coordinates = coordinates.data_array
+            coordinates = coordinates.rename({coordinates.dims[1]: "c"})
+            coordinates = coordinates.assign_coords(dict(c=["x", "y", "z"]))
+            # This is a workaround to remove the warning about stripped units. This line
+            # should be removed once we add/require units in the lcs
+            # Additionally, the correct unit should be checked for TimeSeries
+            # (also expressions)
+            coordinates.data = coordinates.data.m
```

Comment:
See comment

Reply:
I think we have some code that uses units with LCS coords

for now I think the most consistent way in these cases would be, to convert to our 'default' unit of `"mm"`?
This would also raise an exception if units where not of type `"[length]"`

Reply:
It "somehow" works with units, but it isn't inherently supported. Converting to `mm` first is a good suggestion